core/governance/checkpoint.go

```diff
@@ -73,11 +78,16 @@ func (e *Engine) Load(ctx context.Context, data []byte) error {
 
 		switch prop.Terms.Change.GetTermType() {
 		case types.ProposalTermsTypeNewMarket:
+			et := &enactmentTime{
```

Comment:
shouldn't this be moved after line 89? because we're potentially editing the enactment timestamp

Reply:
Cleared it like this https://github.com/vegaprotocol/vega/pull/5668/commits/79f0d5890049572bdf4e3d1052d28c3d9a628d54#diff-92b369672ec5368c6bacbb63f9c3717a133c96f06f473e577ca711abefbcbe23R81